docker/docker-compose.yml

```diff
@@ -2,25 +2,24 @@ version: '3'
 
 services:
   node:
-    image: hyperledger/iroha-docker-develop
+    image: hyperledger/iroha
     ports:
       - "50051:50051"
     environment:
-      - IROHA_POSTGRES_HOST=${COMPOSE_PROJECT_NAME}_postgres_1
+      - IROHA_POSTGRES_HOST=172.18.0.3
```

Comment:
it is very bad to hardcode IPs in docker-compose files. IP can be changed any time. It is better to use name of the service.